website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,212 @@
+
+
+This guide will demonstrate how to: 
+
+- Create a virtual development environment using a template provided by dbt Labs.
+   - This sets up a fully functional dbt environment with an operational and executable project. The codespace automatically connects to the DuckDB database and loads a year's worth of data from our fictional Jaffle Shop café, which sells food and beverages in several US cities.
+   - For additional information, refer to the [README](https://github.com/gwenwindflower/octocatalog) for the Jaffle Shop template. It includes instructions on how to do this, along with animated GIFs.
+- Run any dbt command from the environment’s terminal. 
+- Generate a larger dataset for the Jaffle Shop café (for example, 5 years of data instead of just one).
+
+You can learn [dbt Learn courses](https://learn.getdbt.com) through high-quality courses and workshops.
+
+
+### Related content
+
+
+- [DuckDBsetup](/docs/core/connect-data-platform/duckdb-setup)
+- [Create a GitHub repository](/guides/manual-install?step=2)
+- [Build your first models](/guides/manual-install?step=3)
+- [Test and document your project](/guides/manual-install?step=4)
+- [Schedule a job](/guides/manual-install?step=5)
+
+
+### Prerequisites
+
+- When using DuckDB with dbt Core, you'll need to use the dbt command-line interface (CLI). Currently, DuckDB is not supported in dbt Cloud.
+- It's important that you know some basics of the terminal. In particular, you should understand `cd`, `ls` , and `pwd` to navigate through the directory structure of your computer easily.
+- You have a [GitHub account](https://github.com/join).
+
+## Set up DuckDB for dbt Core
+
+The following will provide a step-by-step guide on setting up DuckDB for use in both local environments and web browsers.
+
+For more information on the setup of DuckDB, you can refer to [DuckDBsetup](/docs/core/connect-data-platform/duckdb-setup).
+
+
+<Tabs>
+
+<TabItem value="local" label="Local">
+
+1. First, you'll need to clone this repository.
+
+    For a more comprehensive understanding and additional information, refer to the [Git Guide](https://github.com/git-guides/git-clone).
+
+    You can access the repository for DuckDB by following the URL:
+
+    ```bash
+    git clone https://github.com.dbt-labs/jaffle_shop_duckdb.git
+
+    ```
+
+2. Change into the docs-duckdb directory from the command line:
+
+    ```Jinja
+
+    cd jaffle_shop_duck_db
+
+    ```
+
+
+3. Install dbt and DuckDB in a virtual environment.
+
+    <Expandable alt_header="Example for MAC" >
+
+    ```Jinja
+
+    python3 -m venv venv
+    source venv/bin/activate
+    python3 -m pip install --upgrade pip
+    python3 -m pip install -r requirements.txt
```

Comment:
At the start of this article, we should point out that the `requirements.txt` file will be referenced to install Core, DuckDB, and every dependency they have. Let them know they can review this file to see everything that will be installed on their machine.